amethyst_core/src/float.rs

```diff
@@ -32,11 +32,22 @@ type FloatBase = f32;
 pub struct Float(FloatBase);
 
 impl Float {
+    /// Returns a new `Float` from a `f64`.
+    pub const fn from_f64(value: f64) -> Self {
+        Float(value)
+    }
+
+    /// Returns a new `Float` from a `f64`.
```

Comment:
```suggestion
    /// Returns a new `Float` from a `f32` using a constant function.
```

Reply:
Same as above. Not necessarily const always.

Reply:
Although in this case it should be `f32` indeed :D

Reply:
oops, fixed that typo!